.vsts-dotnet-ci.yml

```diff
@@ -8,7 +8,7 @@ resources:
     image: microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416
 
 phases:
-- ${{ if ne(variables['codeCoverageBuild'], 'true') }}:
+${{ if ne(variables['codeCoverageBuild'], 'true') }}:
```

Comment:
the - are needed at the beginning of this ifs because phases expect objects to be declared. In yaml you specify an object with `-` the error you're getting I described it in another comment.